tensorflow/lite/experimental/micro/tools/make/targets/mcu_riscv_makefile.inc

```diff
@@ -48,14 +48,30 @@ ifeq ($(TARGET), riscv32_mcu)
 
   MICROLITE_CC_SRCS += \
     $(wildcard tensorflow/lite/experimental/micro/riscv32_mcu/*.cc)
-  MICRO_SPEECH_TEST_SRCS += \
+
+  # Wrap functions
+  MICRO_FE310_LIBWRAP_SRCS += \
     $(wildcard $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/libwrap/sys/*.c) \
-    $(wildcard $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/libwrap/sys/*.cc) \
     $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/libwrap/misc/write_hex.c \
-    $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/libwrap/stdlib/malloc.c \
+    $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/libwrap/stdlib/malloc.c
+
+  MICRO_FE310_BSP_ENV_SRCS += \
     $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/env/start.S \
     $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/env/entry.S \
     $(MAKEFILE_DIR)/downloads/sifive_fe310_lib/bsp/env/freedom-e300-hifive1/init.c
+
+  MICRO_SPEECH_TEST_SRCS += $(MICRO_FE310_LIBWRAP_SRCS) $(MICRO_FE310_BSP_ENV_SRCS)
```

Comment:
Sorry this might be confusing - but we prefer to keep platform specific overrides in the actual example folder. This file should only include the bare-bones parts for building for this platform.

For example, you'll need to create a `riscv32_mcu` folder in each example directory. In those directories, you'll create a new `Makefile.inc` that adds these rules. To give you an example - see this:

The `person_detection` example root Makefile.inc:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/examples/person_detection/Makefile.inc#L52

That Makefile.inc includes platform specifics at L52, such as this one: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/experimental/micro/examples/person_detection/himax_driver/Makefile.inc

Extra source files are added in that new Makefile.inc. 

That would be great if you could do that here - it would help others from getting caught up on Makefile initialization.

Reply:
I have update the patch as you suggest, pls revew if I caught all.
The branch is here https://github.com/zhoupeng/tensorflow/tree/fix_wrapfunction .
Sorry, I don't know if I need to create new PR after update the patch.